src/Illuminate/Foundation/Console/OptimizeClearCommand.php

```diff
@@ -34,7 +34,7 @@ public function handle()
     {
         $this->components->info('Clearing cached bootstrap files.');
 
-        $exceptions = Collection::wrap(explode(',', $this->option('except')))
+        $exceptions = Collection::wrap(explode(',', $this->option('except') ?? ''))
```

Comment:
Is there the need, to call all these at all if `$this->option('except')` is null? Maybe let's just wrap that in an `if` statement

Reply:
see #54214